### PR TITLE
Add pushbullet service

### PR DESCRIPTION
**Pull request by versus**
Added a pushbullet service  (pushbullet.com)

### CODE REVIEW

**Comment by kha7iq** [comment]
Hello @versus Thank you for your contribution great work done.
The library also support SMS with push bullet, can you try to include that.
It will be great to have SMS support with push bullet.
You can use a boolean flag with the cli, for example
```
ping pushbullet --sms true
```
What are your thoughts on this.
Let me know if there is anything i can help with.

**Comment by versus** [comment]
Hello @kha7iq,  I tried doing it before.  
Unfortunately, It doesn't work on my android devices :-(
Maybe,  you can do it,  there is a docs page: https://help.pushbullet.com/articles/how-do-i-send-text-messages-from-my-computer/

I just send alert SMS from opsengine.
By the way,  I think about adding modules for pageduty and opsengine to pingme.

Thanks for awesome tool PingMe. I like to use it.

File: cmd/pushbullet.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	"github.com/nikoksr/notify/service/pushbullet"
+	"github.com/urfave/cli/v2"
+)
+
+// pushBullet struct holds data parsed via flags for pushbullet service.
+type pushBullet struct {
+	Token   string
+	Message string
+	Title   string
+	Device  string
```
Comment (kha7iq) [inline cmd/pushbullet.go:19]:
```
type pushBullet struct {
	Token   string
	Message string
	Title   string
	Device  string
	PhoneNumber string
	SMS bool
}
```
# You an add phone number and SMS boolean flag

**Review by kha7iq — COMMENTED** [review]
let me know if you need some explanation about this.
Please pull the changes from master first and then make the pull request.
I will create an issue for this you can link in pull request

File: cmd/pushbullet.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+				pushBulletOpts.Message,
+			); err != nil {
+				return err
+			}
+
+			log.Println("Successfully sent!")
+			return nil
+		},
+	}
+}
```
Comment (kha7iq) [inline cmd/pushbullet.go:93]:
```
Action: func(ctx *cli.Context) error {
			notifier := notify.New()

			switch pushBulletOpts.SMS {
			case true:
			pushBulletSmsSvc, err := pushbullet.NewSMS(pushBulletOpts.Token, pushBulletOpts.Device)
			if err != nil {
				return err
			}
				devices := strings.Split(pushBulletOpts.PhoneNumber, ",")
				for _, v := range devices {
					if len(v) <= 0 {
						return fmt.Errorf(EmptyChannel)
					}
				pushBulletSmsSvc.AddReceivers(v)

					notifier.UseServices(pushBulletSmsSvc)

					if err := notifier.Send(
						context.Background(),
						pushBulletOpts.Title,
						pushBulletOpts.Message,
					); err != nil {
						return err
					}

				}
			default:
				pushBulletSvc := pushbullet.New(pushBulletOpts.Token)

				devices := strings.Split(pushBulletOpts.Device, ",")
				for _, v := range devices {
					if len(v) <= 0 {
						return fmt.Errorf(EmptyChannel)
					}
					pushBulletSvc.AddReceivers(v)
				}

				notifier.UseServices(pushBulletSvc)

				if err := notifier.Send(
					context.Background(),
					pushBulletOpts.Title,
					pushBulletOpts.Message,
				); err != nil {
					return err
				}

			}

			log.Println("Successfully sent!")
			return nil
		},
	}
}
```

File: cmd/pushbullet.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+				Usage:       "Message content.",
+				EnvVars:     []string{"PUSHBULLET_MESSAGE"},
+			},
+			&cli.StringFlag{
+				Destination: &pushBulletOpts.Title,
+				Name:        "title",
+				Value:       TimeValue,
+				Usage:       "Title of the message.",
+				EnvVars:     []string{"PUSHBULLET_TITLE"},
+			},
```
Comment (kha7iq) [inline cmd/pushbullet.go:65]:
```
	&cli.BoolFlag{
				Destination: &pushBulletOpts.SMS,
				Name:        "sms",
				Value:       false,
				Usage:       "To send sms message set the value to 'true'",
				EnvVars:     []string{"PUSHBULLET_SMS"},
			},
		},

&cli.StringFlag{
				Destination: &pushBulletOpts.PhoneNumber,
				Name:        "number",
				Aliases:     []string{"n"},
				Usage:       "Device's nickname of pushbullet.",
				EnvVars:     []string{"PUSHBULLET_NUMBER"},
			},
```

**Review by kha7iq — CHANGES_REQUESTED** [review]
these changes are required to send sms with a boolean flag

**Comment by kha7iq** [comment]
@ Do let me know today , if you can include these change, if no can you please pull the changes from master before making the pull request so that i can merge existing contribution and include the rest my self.

**Comment by kha7iq** [comment]
Closing this as stale, i will include you contribution and open a separate PR

**Pull request closed without merging**